packages/vscode-extension/README.md

```diff
@@ -0,0 +1,77 @@
+# moon README
```

Comment:
The README needs to be altered to reflect the VSCode extension's purpose as it would be displayed on the marketplaces.